dockerfiles/Dockerfile.openresty

```diff
@@ -67,7 +67,7 @@ RUN if [ "$EDITION" = "enterprise" ] ; then curl -fsSLo /tmp/passwdqc-${KONG_DEP
 
 ARG KONG_NGINX_MODULE=master
 
-ARG RESTY_VERSION=1.13.6.2
+ARG RESTY_VERSION=1.17.8.2
```

Comment:
Using 1.17.8.2 required by kong  next branch.  The next branch is required to support not using "--with-pcre-jit" with OpenResty because pcre does not build on Z.